src/ModelContextProtocol/Shared/McpSession.cs

```diff
@@ -16,6 +18,13 @@ namespace ModelContextProtocol.Shared;
 /// </summary>
 internal sealed class McpSession : IDisposable
 {
+    private static readonly ActivitySource s_activitySource = new("ModelContextProtocol");
+    private static readonly Meter s_meter = new("ModelContextProtocol");
+    private static readonly Histogram<double> s_operationDurationHistogram = s_meter.CreateHistogram<double>(
+        "modelcontextprotocol.operation.duration",
+        "s",
+        "Measures the duration of an operation in seconds.");
```

Comment:
`duration` seems to be the most important metric for any kind of RPC mechanism. It'd probably be good to add a similar counter for the entire session's duration. Are there any other high-priority meters?

Reply:
This sounds like the corollary to request duration and connection duration for http.

Reply:
> It'd probably be good to add a similar counter for the entire session's duration.

Ok